CHANGELOG.md

```diff
@@ -2,9 +2,13 @@
 
 ## 0.8.2 — [unreleased]
 
+_The following unreleased version are only available in the `main` branch. They
```

Comment:
This had seemed implicit from "Unreleased", but maybe that's becaues I'm used to it...

Reply:
I added this sentence to make it clear where someone _could_ find these features (in the `main` branch), even though they're not in any stable branch.

And I'm treating this PR as an experiment: I wanted to see how little information was required of a developer to leave hints about the unreleased stuff. 

I'm willing to be fairly forgiving in the format of the Unreleased section: a one-liner that points to a PR, an issue, or even a discussion (github or discord) that talks about what has actually been implemented is useful. Thanks.